### PR TITLE
Add remove items

### PR DESCRIPTION
**Pull request by Cathella**
- [ ] Remove hard-coded items from the tasks array.
- [ ] Add a JavaScript file for the new functionality.
- [ ] Implement a function for adding a new task.
- [ ] Implement a function for editing task descriptions..
- [ ] All changes to the To-Do List are saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,31 +1,57 @@
...
+if (document.querySelectorAll('edit-button')) {
+  const editButtons = document.querySelectorAll('.edit-button');
+
+  editButtons.forEach((btn) => {
+    btn.addEventListener('click', () => {
+      const editingTask = btn.parentNode;
+      editingTask.classList.add('editing');
+    });
+  });
+}
```
Comment (rukundoeric) [inline src/index.js:55]: - [ ] <b>REQUIRED:</b> Your function is adding new item in the list after duplicating the current items. kindly fix it.
scrnli_10_3_2021_12-29-55 AM

File: src/index.html [hunk 2]
```diff
@@ -8,9 +8,24 @@
...
+      </header>
+      <div class="form">
+        <input type="text" id="input-task" placeholder="Add to your list ...">
+        <button type="submit" id="add-button">
+           + 
+        </button>
+      </div>
       <!-- Create an empty To Do List placeholder -->
       <div id="todo-list"></div>
+      <button type="button" id="clear-button">Clear all completed</button>
```
Comment (rukundoeric) [inline src/index.html:23]: - [ ] <b>REQUIRED:</b> Clear all functionality is not implemented, kindly do that to comply with the project specifications.
- [ ] - [ ] <b>REQUIRED:</b> Edit task functionality is not implemented, kindly do that to comply with the project specifications.
- [ ] - [ ] - [ ] <b>REQUIRED:</b> Deleting a  single task functionality is not implemented, kindly do that to comply with the project specifications.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Cathella, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Cathella, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Editing task feature is not implemented. Kindly implement it to comply with the project specifications.
  scrnli_10_3_2021_6-10-27 AM


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Cathella** [comment]
Hi @rukundoeric , thank you for the review, however the update/edit functionality has already been implemented as I can show you in the loom link - [https://www.loom.com/share/a8eae7ea64724c0685cf723bdab99e0b](url).
live link - [https://raw.githack.com/Cathella/todo-app/add-remove-items/dist/index.html](url).

It would unfair repeating this week because of this.

**Pull request merged**